src/objective-c/GRPCClient/private/GRPCCompletionQueue.m

```diff
@@ -66,30 +66,17 @@ - (instancetype)init {
       while (YES) {
         // The following call blocks until an event is available.
         grpc_event *event = grpc_completion_queue_next(unmanagedQueue, gpr_inf_future);
+        GRPCQueueCompletionHandler handler;
```

Comment:
Argh, right, I now remember it bit me in the previous implementation of this method.